app/models/event_stream.rb

```diff
@@ -22,4 +22,6 @@ class EventStream < ApplicationRecord
   belongs_to :container_replicator
   belongs_to :container_group
   belongs_to :container_node
+
+  belongs_to :middleware_server, :foreign_key => :middleware_server_id
```

Comment:
I've remembered it. Yes it is needed as the fk comes from the event_streams table which was updated in a previous work with the middleware_server_id column as a foreign key.